core/lib/zksync_core/src/sync_layer/gossip/storage/mod.rs

```diff
@@ -116,22 +213,28 @@ impl PostgresBlockStorage {
             .map_err(StorageError::Database)
     }
 
+    async fn sync_block(
+        ctx: &ctx::Ctx,
+        storage: &mut StorageProcessor<'_>,
+        number: MiniblockNumber,
+    ) -> StorageResult<Option<SyncBlock>> {
+        let operator_address = Address::default(); // FIXME: where to get this address from?
```

Comment:
Just to highlight this remnant from the previous PR: I don't know where this address can be obtained from. It's used in case a miniblock is not sealed.

Reply:
@brunoffranca  what was the long term plan on how to deal with operator address? AFAIR it is currently fixed anyway, but eventually will be dynamic.

Reply:
JFYI, I've created a separate task in Linear to deal with the operator address.